src/Cryptol/Eval/Value.hs

```diff
@@ -22,10 +22,19 @@
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
Minor style suggestion: it would be worth alphabetizing this `LANGUAGE` pragma with respect to the other pragmas. (Similarly for all other places where `LANGUAGE PatternSynonyms` is introduced in this patch.)